src/taipy/core/config/config.schema.json

```diff
@@ -12,6 +12,10 @@
       "required": [],
       "additionalProperties": {
         "properties": {
+          "core_version": {
```

Comment:
In fact the whole CORE description is wrong:
additionalProperties.properties.* should be in properties

CORE is not like DATA_NODE: it doesn't have an intermediate level